integration_tests/integration/confirmAddedToRoll.spec.ts

```diff
@@ -29,6 +30,10 @@ context('Confirm Added To Roll', () => {
     const confirmArrivalPage = ConfirmArrivalPage.goTo(expectedArrival.id)
     confirmArrivalPage.continue().click()
 
+    const sexPage = SexPage.goTo(expectedArrival.id) as SexPage
+    sexPage.sexRadioButtons('F').click()
+    sexPage.continue().click()
```

Comment:
Probably better to not explicitly test this functionality in other tests.. could just ensure that the test data has a gender in BASM.

Reply:
change ImprisonmentPage go to - to go via the sex page ??